TerminalDocs/customize-settings/interaction.md

```diff
@@ -0,0 +1,170 @@
+
+<br />
+
+___
+
+## Snap window resizing to character grid
+
+:::row:::
+:::column span="":::
+When this is set to `true`, the window will snap to the nearest character boundary on resize. When it's set to `false`, the window will resize "smoothly".
+
+**Property name:** `snapToGridOnResize`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `true`
+
+:::column-end:::
+:::column span="":::
+![Windows Terminal snap to grid on resize](./../images/snap-to-grid-on-resize.gif)
+
+:::column-end:::
+:::row-end:::
+
+<br />
+
+___
+
+## Tab settings
+
+### Tab switcher interface style
+
+:::row:::
+:::column span="":::
+When this is set to `true` or `"mru"`, the `nextTab` and `prevTab` commands will use the tab switcher UI, with most recently used ordering. When set to `"inOrder"`, these actions will switch tabs in their current order in the tab bar. The UI will show all the currently open tabs in a vertical list, navigable with the keyboard or mouse.
+
+The tab switcher will open on the initial press of the actions for `nextTab` and `prevTab`, and will stay open as long as a modifier key is held down. When all modifier keys are released, the switcher will close and the highlighted tab will be focused. <kbd>tab</kbd>/<kbd>shift+tab</kbd>, the <kbd>up</kbd> and <kbd>down</kbd> arrow keys, and the `nextTab`/`prevTab` actions can be used to cycle through the switcher UI.
+
+To disable the tab switcher, you can set this to `false` or `"disabled"`.
+
+**Property name:** `tabSwitcherMode`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`, `"mru"`, `"inOrder"`, `"disabled"`
+
+**Default value:** `"inOrder"`
+
+:::column-end:::
+:::column span="":::
+![Windows Terminal tab switcher](./../images/tab-switcher.gif)
+
+:::column-end:::
+:::row-end:::
+
+### Enable tab switcher
+
+When this is set to `true`, the `nextTab` and `prevTab` commands will use the tab switcher UI. The UI will show all the currently open tabs in a vertical list, navigable with the keyboard or mouse.
+
+The tab switcher will open on the initial press of the actions for `nextTab` and `prevTab`, and will stay open as long as a modifier key is held down. When all modifier keys are released, the switcher will close and the highlighted tab will be focused. <kbd>tab</kbd>/<kbd>shift+tab</kbd>, the <kbd>up</kbd> and <kbd>down</kbd> arrow keys, and the `nextTab`/`prevTab` actions can be used to cycle through the switcher UI.
+
+**Property name:** `useTabSwitcher`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `true`
+
+> [!CAUTION]
+> The `"useTabSwitcher"` setting is no longer available in versions 1.5 and later. It is recommended that you use the `"tabSwitcherMode"` setting instead.
+
+<br />
+
+___
+
+## Paste warnings
```

Comment:
Add a tip/warning that these are not accessible via SUI?

Reply:
Pushing this off until the settings UI becomes default.